keylime/ca_util.py

```diff
@@ -151,6 +152,20 @@ def cmd_mkcert(workingdir: str, name: str, password: Optional[str] = None) -> No
         os.chdir(cwd)
 
 
+def cmd_import_priv(priv_pem_file: str, lastserial: int) -> None:
```

Comment:
This function seems to need workingdir as a parameter and then you have to ch_dir to this directory otherwise the key goes into an unexpected place.

Reply:
added